### PR TITLE
Dockerfile with ubi8-minimal as base image

### PR DESCRIPTION
**Pull request by prakhar-aqua**
This PR contains Dockerfile.ubi8 which uses ubi8-minimal as base image for Kube-bench

### CODE REVIEW

File: Dockerfile.ubi8 [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+# ubi8-minimal base image for build with ubi standards
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5 as run
+
+RUN RUN microdnf install yum \
+  && yum -y update-minimal --security --sec-severity=Moderate --sec-severity=Important --sec-severity=Critical \
+  && yum clean all \
+  && yum install glibc \
+  && yum update -y glibc \
```
Comment (mozillazg) [inline Dockerfile.ubi8:20]: Could please include the `jq` package, the `procps` package and override the `PATH` env?:

https://github.com/aquasecurity/kube-bench/blob/d190cbcaf66d2681c8782081f51c3e9afaef5eb4/Dockerfile#L14-L16

https://github.com/aquasecurity/kube-bench/blob/d190cbcaf66d2681c8782081f51c3e9afaef5eb4/Dockerfile#L31-L33

**Review by mozillazg — CHANGES_REQUESTED** [review]

File: Dockerfile.ubi8 (hunk 1 above)
Comment (prakhar-aqua) [reply Dockerfile.ubi8:20]: Done

File: Dockerfile.ubi8 [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+RUN make build && cp kube-bench /go/bin/kube-bench
+
+
+# ubi8-minimal base image for build with ubi standards
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5 as run
+
+RUN microdnf install yum \
+  && yum -y update-minimal --security --sec-severity=Moderate --sec-severity=Important --sec-severity=Critical \
+  && yum update -y \
+  && yum install glibc \
```
Comment (mozillazg) [inline Dockerfile.ubi8:19]:
```suggestion
  && yum install -y glibc \
```

File: Dockerfile.ubi8 [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  && yum update -y glibc \
+  && yum install procps \
```
Comment (mozillazg) [inline Dockerfile.ubi8:21]:
```suggestion
  && yum install -y procps \
```

File: Dockerfile.ubi8 [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  && yum update -y procps \
+  && yum install epel-release -y \
```
Comment (mozillazg) [inline Dockerfile.ubi8:23]: this command will cause an error:

```
#7 101.7 No match for argument: epel-release
#7 101.7 Error: Unable to find a match: epel-release
```
```suggestion
```

File: Dockerfile.ubi8 [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+COPY cmd/ cmd/
+COPY internal/ internal/
+ARG KUBEBENCH_VERSION
+RUN make build && cp kube-bench /go/bin/kube-bench
+
+
+# ubi8-minimal base image for build with ubi standards
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5 as run
+
+RUN microdnf install yum \
```
Comment (mozillazg) [inline Dockerfile.ubi8:16]: `findutils` is needed for having `find` commands and`openssl` is needed for having `openssl` commands

```suggestion
RUN microdnf install yum findutils openssl \
```

**Review by mozillazg — CHANGES_REQUESTED** [review]

File: Dockerfile.ubi8 (hunk 2 above)
Comment (prakhar-aqua) [reply Dockerfile.ubi8]: added -y

File: Dockerfile.ubi8 (hunk 3 above)
Comment (prakhar-aqua) [reply Dockerfile.ubi8]: added -y

File: Dockerfile.ubi8 (hunk 4 above)
Comment (prakhar-aqua) [reply Dockerfile.ubi8]: Removed epel-release package

File: Dockerfile.ubi8 (hunk 5 above)
Comment (prakhar-aqua) [reply Dockerfile.ubi8]: added findutils and openssl

**Review by mozillazg — APPROVED** [review]
LGTM!

**Comment by prakhar-aqua** [comment]
> @prakhar-aqua LGTM as well , please rebase your branch with upstream

rebase done

**Pull request merged**